lib/workers/repository/update/branch/index.ts

```diff
@@ -333,7 +333,7 @@ export async function processBranch(
     ) {
       // Only set a stability status check if one or more of the updates contain
       // both a minimumReleaseAge setting and a releaseTimestamp
-      config.stabilityStatus = 'green';
+      config.ageStatus = 'green';
```

Comment:
was it a templateable field? so it's breaking there?

Reply:
No, it's not